src/Internal/Serialization/Keys.js

```diff
@@ -0,0 +1,16 @@
+/* global BROWSER_RUNTIME */
+
+const bytesFromKey = key => key.as_bytes();
+
+exports.bytesFromPublicKey = bytesFromKey;
+exports.bytesFromPrivateKey = bytesFromKey;
+
+exports.publicKeyFromPrivateKey = private_key => () => {
+  return private_key.to_public();
```

Comment:
No need for an explicit `return` since there's just one line, you can remove the keyword and the brackets to implicitly return the express (`private_key.to_public()`)

Reply:
Looks like it does not need to be effect-full in the first place. Was there a reason for its original effect-full implementation or should I refactor it to be pure?

Reply:
I think it can be pure